project-base/tests/App/Acceptance/acceptance/CartCest.php

```diff
@@ -307,4 +293,23 @@ public function testPromoCodeFlowInCart(
         $cartPage->canSeePromoCodeSubmitButtonElement();
         $cartPage->assertTotalPriceWithVatByProducts($products);
     }
+
+    /**
+     * @param \Tests\App\Test\Codeception\AcceptanceTester $me
+     * @param string $productName
+     * @param int $quantity
+     */
+    protected function seeRightMessageForAddedProducts(AcceptanceTester $me, string $productName, int $quantity): void
```

Comment:
_nitpick_ 🤓 

What is _the right_ message? (also, it checks only for one product, not products)
What about `seeSuccessMessageForAddedProduct`?

Reply:
Done